looker/resource_looker_user_attribute_user_test.go

```diff
@@ -0,0 +1,134 @@
+package looker
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	sdk "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+	"github.com/pkg/errors"
+	"github.com/resolutionlife/terraform-provider-looker/internal/conv"
+	"github.com/resolutionlife/terraform-provider-looker/internal/slice"
+)
+
+func init() {
+	// Add a sweeper to remove user attributes and users that have names starting with `test_acc`.
+	resource.AddTestSweepers("looker_user_attribute_user", &resource.Sweeper{
+		Name: "looker_user_attribute_user",
+		F: func(r string) error {
+			c, err := newTestLookerSDK()
+			if err != nil {
+				return err
+			}
+
+			userAttrs, err := c.AllUserAttributes(sdk.RequestAllBoardSections{
+				Fields: conv.P(""),
+				Sorts:  conv.P(""),
+			}, nil)
+			if err != nil {
+				return err
+			}
+
+			filteredUserAttrs := make([]sdk.UserAttribute, 0, len(userAttrs))
+			for _, userAttr := range userAttrs {
+				if strings.HasPrefix(userAttr.Name, "test_acc_") {
+					filteredUserAttrs = append(filteredUserAttrs, userAttr)
+				}
+			}
+
+			for _, filteredUserAttr := range filteredUserAttrs {
+				if _, err := c.DeleteUserAttribute(*filteredUserAttr.Id, nil); err != nil {
+					return err
+				}
+			}
+
+			users, err := c.SearchUsers(sdk.RequestSearchUsers{
+				Email: conv.P("test-acc%"),
+			}, nil)
+			if err != nil {
+				return err
+			}
+
+			for _, u := range users {
+				if _, err := c.DeleteUser(*u.Id, nil); err != nil {
+					return err
+				}
+			}
```

Comment:
We already have a sweeper for similar users. Could we maybe use the `Dependencies` slice attribute on `resource.Sweeper`?

Reply:
That's a very useful resource and works really well, might have to back and check if any of the other tests can benefit from this too.